crates/wysiwyg/src/dom/dom_struct.rs

```diff
@@ -388,6 +388,83 @@ where
             panic!("Parent node was not a container!");
         }
     }
+
+    /// Checks if the passed [handle] exists in the DOM.
+    pub fn exists(&self, handle: &DomHandle) -> bool {
```

Comment:
I think this checks if the DOM contains an element. Should this method then be called `contains`?